src/Core/Components/Slider/FluentSlider.razor

```diff
@@ -0,0 +1,28 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentInputBase<TValue>
+@typeparam TValue
+
+<FluentField InputComponent="@this" ForId="@Id" Class="@ClassValue" Style="@StyleValue">
+	<fluent-slider slot="input"
+	               id="@Id"
+	               disabled="@Disabled"
+	               min="@FormatValueAsStringOrNull(Min)"
+	               max="@FormatValueAsStringOrNull(Max)"
+	               value="@FormatValueAsStringOrNull(Value)"
+	               step="@FormatValueAsStringOrNull(Step)"
+	               size="@Size.ToAttributeValue()"
+	               orientation="@(Orientation == Orientation.Horizontal ? null : Orientation.ToAttributeValue())"
+	               mode="@Mode.ToAttributeValue()"
+	               name="@Name"
+	               readonly="@ReadOnly"
+	               aria-label="@AriaLabel"
+	               autofocus="@Autofocus"
+	               required="@Required"
+	               tabindex="@GetTabIndexValue"
```

Comment:
Either you use the verb "get" and that must be a method with parenthesis: `GetTabIndexValue()`
Either you don't use a verb and that must be a property `TabIndexValue`

Reply:
PS: I prefer to use a method :-)